int32 block_index
---
int32 assembly_pad_index
#
int32 size_x
int32 size_y
string color
#
int32 position_x
int32 position_y
int32 position_z
#
bool need_force_limit
int32 shift_x
int32 shift_y
int32 position_z_max
